### DISCUSSION TITLE
Rename `uncertainty_threshold` in ensemble to uncertainty?

### DISCUSSION THREAD

**Discussion by bagustris** [felixbur/nkululeko#144] [General]
Since `uncertainty_threshold` can be used as both "uncertainty lowest" (when no `--threshold` arg is given) and "uncertainty threshold" (with `--threshold` arg), I think it is better to rename it to "uncertainty" only to make it more general name. What do you think @felixbur ?

**Comment by felixbur** [comment]
fine with me
